File: go-client-sdk/pkg/models/shared/stringlistfilter.go
package shared

type StringListFilter struct {
	Has      *string  `json:"has,omitempty"`
	HasEvery []string `json:"hasEvery,omitempty"`
	HasSome  []string `json:"hasSome,omitempty"`
	IsEmpty  *bool    `json:"isEmpty,omitempty"`
}

File: typescript-client-sdk/src/sdk/models/shared/stringlistfilter.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";



export class StringListFilter extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "json, name=has" })
  has?: string;

  @SpeakeasyMetadata({ data: "json, name=hasEvery" })
  hasEvery?: string[];

  @SpeakeasyMetadata({ data: "json, name=hasSome" })
  hasSome?: string[];

  @SpeakeasyMetadata({ data: "json, name=isEmpty" })
  isEmpty?: boolean;
}
